lib/generators/shopify_app/home_controller/templates/index.html.erb

```diff
@@ -9,6 +9,10 @@
       />
 <% unless with_cookie_authentication? %>    <script>
       document.addEventListener("DOMContentLoaded", async function() {
+        <% if ShopifyApp.use_importmap? %>
+          await import("lib/shopify_app")
```

Comment:
Is this the best way to load  `shopify_app.js`? It requires the DOM content to be loaded, but we need to run it before this point.